src/reformatters/example/dynamical_dataset.py

```diff
@@ -0,0 +1,51 @@
+from collections.abc import Sequence
+
+from reformatters.common import validation
+from reformatters.common.dynamical_dataset import DynamicalDataset
+from reformatters.common.kubernetes import Job
+
+from .region_job import ExampleRegionJob, ExampleSourceFileCoord
+from .template_config import ExampleDataVar, ExampleTemplateConfig
+
+
+class ExampleDataset(DynamicalDataset[ExampleDataVar, ExampleSourceFileCoord]):
+    template_config: ExampleTemplateConfig = ExampleTemplateConfig()
+    region_job_class: type[ExampleRegionJob] = ExampleRegionJob
+
+    def operational_kubernetes_resources(self, image_tag: str) -> Sequence[Job]:
+        """Return the kubernetes cron job definitions to operationally update and validate this dataset."""
+        # operational_update_cron_job = ReformatCronJob(
+        #     name=f"{self.dataset_id}-operational-update",
+        #     schedule=_OPERATIONAL_CRON_SCHEDULE,
+        #     pod_active_deadline=timedelta(minutes=30),
+        #     image=image_tag,
+        #     dataset"_id=self.dataset_id,
+        #     cpu="14",
+        #     memory="30G",
+        #     shared_memory="12G",
+        #     ephemeral_storage="30G",
+        # )
+        # validation_cron_job = ValidationCronJob(
+        #     name=f"{self.dataset_id}-validation",
+        #     schedule=_VALIDATION_CRON_SCHEDULE,
+        #     pod_active_deadline=timedelta(minutes=10),
+        #     image=image_tag,
+        #     dataset_id=self.dataset_id,
+        #     cpu="1.3",
+        #     memory="7G",
+        # )
+
+        # return [operational_update_cron_job, validation_cron_job]
+        raise NotImplementedError(
+            f"Implement `operational_kubernetes_resources` on {self.__class__.__name__}"
+        )
+
+    def validators(self) -> Sequence[validation.DataValidator]:
+        """Return a sequence of DataValidators to run on this dataset."""
+        # return (
+        #     validation.check_analysis_current_data,
+        #     validation.check_analysis_recent_nans,
+        # )
```

Comment:
I wonder if we should avoid suggesting using these validators or pulling from common vs the SWANN approach of having a dataset specific `validators.py` where these are defined. The validations in `common` didn't really fit for SWANN. Should we add `validators` to our `example/` directory?